src/components/Button/Button.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { openModal } from './../../actions/index';
+
+import './Button.scss';
+
+const Button = ({ type, icon, children, link, modal, click, disabled }) => {
```

Comment:
you can just pass onClick functionality from parent so then we dont need the 2 diff buttons instead you can just pass the functionality declared on the parent. 

example: 
```
<button className='button' type={type} onClick={onClick} disabled={disabled}>{plusIcon}{children}</button>;
```

Reply:
We should also pass `ref` from parent to be safe

Reply:
Then you can remove the dispatch too

Reply:
Thanks for the suggestions!